netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -184,14 +183,13 @@ public void cancelBufferedStreamShouldChangeClientStreamStatus() throws Exceptio
 
     assertTrue(createPromise.isSuccess());
     verify(stream).transportReportStatus(eq(Status.CANCELLED), eq(true),
-        any(Metadata.Trailers.class));
+            any(Metadata.Trailers.class));
```

Comment:
The problem is that each IDE (even version) formats it differently, so you get constant churn. The constant useless churn that adds noise is what I have issue with. I'm thinking there should be a ban on full-file reformatting.

As a further example, the newline between the io.grpc and io.netty imports is minor but also improper, and so later we will have another PR that will fix it. The only way to stop the cycle it to stop committing useless code reformatting like here.